app/assets/stylesheets/users.scss

```diff
@@ -8,6 +8,7 @@
 }
 
 #body {
+    font-family: Corbel, "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", "DejaVu Sans", "Bitstream Vera Sans", "Liberation Sans", Verdana, "Verdana Ref", sans-serif;
```

Comment:
This isn't needed, as it's specified for the entire `body` element. All child elements inherit from `body`.  Same as with line 36.